frontend/tests/.server/domain/services/notify-service-default.test.ts

```diff
@@ -0,0 +1,133 @@
+import { describe, expect, it, vi, beforeEach } from 'vitest';
+
+import type {
+  NotificationEmailRequest,
+  NotificationEmailResponseOK,
+  NotificationEmailResponseError,
+} from '~/.server/domain/models';
+import { getDefaultNotificationService } from '~/.server/domain/services/notification-service-default';
+import { serverEnvironment } from '~/.server/environment';
+import { HttpStatusCodes } from '~/errors/http-status-codes';
+
+// Mock the global fetch
+const mockFetch = vi.fn();
+global.fetch = mockFetch;
+
+describe('notification-service-default', () => {
+  //
+  const notificationService = getDefaultNotificationService();
+
+  const notifyEmail = 'simulate-delivered@notification.canada.ca';
+  const notifyEngEmail = `${serverEnvironment.GC_NOTIFY_ENGLISH_TEMPLATE_ID}`;
+  const notifyFrEmail = `${serverEnvironment.GC_NOTIFY_FRENCH_TEMPLATE_ID}`;
```

Comment:
```suggestion
  const notifyEngEmail = serverEnvironment.GC_NOTIFY_ENGLISH_TEMPLATE_ID;
  const notifyFrEmail = serverEnvironment.GC_NOTIFY_FRENCH_TEMPLATE_ID;
```

Also, the variable names don't really indicate that they are template IDs.. I'd recommend using something like `enTemplateId` and `frTemplateId`.